<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displayCfgDialog</class>
 <widget class="QDialog" name="displayCfgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>显示器模式设置</string>
  </property>
  <widget class="QListWidget" name="listWidget_res">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1920 * 1080 60hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280 * 720 60hz</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>351</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 * 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 * 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 * 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 * 3</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_monitor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>monitor 1 dp-7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>monitor 2 dp-4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_monitors">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>所有显示器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resolutions">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>分辨率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_modes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="displayCfgButtonBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>536</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>提示:单击鼠标修改显示器</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>displayCfgButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>displayCfgDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayCfgButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>displayCfgDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
